operations/update.go

```diff
@@ -59,21 +59,28 @@ func Update() cli.Command {
 			if err != nil {
 				return errors.Wrap(err, "problem loading configuration")
 			}
-			if conf.AutoUpgradeCLI && autoUpdate {
-				fmt.Printf("automatic CLI updates are already set up, %s flag is not necessary", autoUpdateFlagName)
+			if !conf.AutoUpgradeCLI && !autoUpgrade {
+				fmt.Printf("Automatic CLI upgrades are not set up, specifying the -%s (-a for short) flag will enable automatic CLI upgrades before each command if the current CLI is out of date.", autoUpgradeFlagName)
```

Comment:
some nits: I would remove the -a for short; since it's something they only need to do once a shorthand isn't even super necessary (I would even argue against using it at all in case we want to use -a for something more common in the future). Also can you use a semicolon before "specifying" since that's the start of a new sentence?

Reply:
True, I'll just remove -a completely.